packages/lib/src/editors/Cropper.wc.svelte

```diff
@@ -0,0 +1,186 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import 'cropperjs/dist/cropper.css';
```

Comment:
Importing this path is not the correct way to import cropper since it references the 'dist' folder. Cropper should be implemented like the other editors, without any additional installations of modules.